### ISSUE TITLE
So does this actually work?

### ISSUE THREAD

**Issue by zorgrian** [afarhan/antuino2.2#1]
If not then why not?

Is it fully tested?

## Background 
Ashhar Farhan12:37pm   #79891  
I have been using the Antuino exclusively as my main instrument for two weeks now. In the process, I have had to rewrite the entire firmware to make it more usable.
I have uploaded the new code.  I only recommend it for the fool hardy. It is available on github.com/afarhan/antuino2.2
Among the new features is continuous scanning at very fast sweeps, continuous coverage with no gaps to 250 mhz, and much simpler user interface. I have also added a voltmeter hack onto the extra pin A&. You have to add the attached circuitry to it.

**Comment by afarhan** [comment]
no, we are still testing it. if you know how to upload it properly, then you can use it. if it doesn't work you should know how to go back to the earlier version from github
